Add VPE to voice product docs (#37513)
home-assistant/home-assistant.github.io: source/voice_control/index.markdown

Assist allows you to control Home Assistant using natural language. It is built on top of an open voice foundation and powered by knowledge provided by our community.

The simplest way to try out Assist is inside our companion app. Look for the Assist icon <img src='/images/assist/assist-icon.svg' alt='Assist icon' style='height: 32px' class='no-shadow'> at the top right of your dashboard.

The simplest way to get started with Assist is with our recommended voice assistant hardware, the [Home Assistant Voice Preview Edition](/voice-pe/).

As for the rest of Home Assistant core functionalities, Assist can be personalized and extended to fit your needs.

- It can work locally or leverage the greatest LLMs of the moment.
- It can work on your phone or tablet or other custom voice devices.

<lite-youtube videoid="XF53wUbeLxA" videotitle="Voice at Home Assistant"></lite-youtube>

## Getting Started

When you configure voice assistant hardware made for Home Assistant, it will use a wizard to help you configure your system and get started to use voice.

Our recommended voice assistant hardware is the [Home Assistant Voice Preview Edition](/voice-pe/).

> In case your hardware does not support our wizard, do not worry. Here are two detailed guides based on how you plan to process your voice (Locally, or using Home Assistant Cloud voice services)

Reviewer: how do I know if my hardware supports the wizard?

Assist allows you to control Home Assistant using natural language. It is built on top of an open voice foundation and powered by knowledge provided by our community.

> Assist is available to use on most platforms that can interface with Home Assistant. Look for the Assist icon <img src='/images/assist/assist-icon.svg' alt='Assist icon' style='height: 32px' class='no-shadow'>

Reviewer 2: We have a lot of single sentences as paragraphs. I would glue this one to previous paragraph.

> The simplest way to try out Assist is side our companion app. We have dedicated pages to guide you on both [Android](/voice_control/android/) and [iOS](/voice_control/apple/).

Reviewer 2: I have actually mixed feelings on this tip, because users don't have STT and TTS configured at this point.

I also think that this sentence can be simplified, you don't need a guide just to find a button. And then it becomes the same sentence as the one above. Let's combine them.

```suggestion
The simplest way to try out Assist is inside our companion app. Look for the Assist icon <img src='/images/assist/assist-icon.svg' alt='Assist icon' style='height: 32px' class='no-shadow'> at the top right of your dashboard.
```

Author: okay!

The simplest way to get started with Assist is with our recommended voice assistant hardware, the [Home Assistant Voice Preview Edition](/voice-pe/).

As for the rest of Home Assistant core functionalities, Assist can be personalized and extended to fit your needs.

- It can work locally or leverage the greatest LLMs of the moment.
- It can work on your phone or tablet or other custom voice devices.

<lite-youtube videoid="XF53wUbeLxA" videotitle="Voice at Home Assistant"></lite-youtube>

## Getting Started

Most of the recent voice assistant hardware compatible with Home Assistant will automatically guide you thought their configuration by utilizing a wizard.

> ![Screenshot of Home Assistant's voice wizard](/images/assist/voice-wizard.png)

Reviewer 2: This picture seems very big, and it adds no value to the text.

Author: I do not think it hurt either, right?
I simply wanted to show users what the "wizard" looks like so that they know what to expect

> Most of the recent voice assistant hardware compatible with Home Assistant will automatically guide you thought their configuration by utilizing a wizard.
> 
> ![Screenshot of Home Assistant's voice wizard](/images/assist/voice-wizard.png)
> 
> All you need to do is integrate the device into your Home Assistant installation, and follow the steps displayed on the screen.
> The wizard will help you get stated with your hardware based on your system.

Reviewer 2: ```suggestion
When you configure voice assistant hardware made for Home Assistant, it will use a wizard to help you configure your system and get started to use voice.
```

Author: ok for removing the picture 😊
